src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
GitHub's complaining here about there being no newline at the end of the line.
If adding a LF character to the end of the file won't break anything, please do.